OverlayPlugin.Core/MemoryProcessors/AtkGui/FFXIVClientStructs/Global/FFXIV/Client/Game/ActionManager.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Runtime.InteropServices;
+using RainbowMage.OverlayPlugin.MemoryProcessors.AtkStage.FFXIVClientStructs.Global.FFXIV.Client.Game.Object;
+using RainbowMage.OverlayPlugin.MemoryProcessors.AtkStage.FFXIVClientStructs.Global.FFXIV.Client.Graphics;
+namespace RainbowMage.OverlayPlugin.MemoryProcessors.AtkStage.FFXIVClientStructs.Global.FFXIV.Client.Game
+{
+
+    [StructLayout(LayoutKind.Explicit, Size = 0x810)]
+    public unsafe struct ActionManager
+    {
+        [FieldOffset(0x13C)] public fixed uint BlueMageActions[24];
+
+
+
+
+
+
+
```

Comment:
?

Reply:
I did minimal styling cleanup on any of the `FFXIVClientStructs/Global/*` files (only what `dotnet format` does by default).

These files should be cleaned up before merge, but I didn't want to waste the effort of cleaning up every file if ultimately we're only going to pull in what's required for `AddonPartyList` and `AtkStage` in this PR.